pages.pt_BR/linux/csplit.md

```diff
@@ -22,4 +22,4 @@
 
 - Dividir um arquivo na linha que atenda a expressão regular:
 
-`csplit {{arquivo}} /{{regex}}/`
+`csplit {{arquivo}} /{{expressão_regular}}/`
```

Comment:
in the Spanish translations we remove the accents, like when we program in Spanish on any programming language.
I think we should do the same here:
```suggestion
`csplit {{arquivo}} /{{expressao_regular}}/`
```

Reply:
Sounds fine, but I'd like to open an issue where we can discuss this.  Personally, I don't see a strong reason to remove the accents.

Reply:
That's why your language doesn't have accents :joy:
If I see an `é`, `ó` or `ñ` I don't think in that word like a "variable".

Reply:
@navarroaxel should this also be done in `pt_PT` pages?

Reply:
Hrm. Are there any other spanish speakers here we can ask for an opinion?

Reply:
There haven't been contributions to the `pt_BR` or `pt_PT` directory for a long time, so I don't think so.

Reply:
Also the Turkish pages, e.g. https://github.com/tldr-pages/tldr/blob/a21ae3f3996fdd943b0dd3468f0ffa3b331d21c8/pages.tr/linux/xterm.md.

Reply:
cc @ivanhercaz